Parse/src/main/java/com/parse/Parse.java

```diff
@@ -158,72 +152,73 @@ public Builder server(String server) {
       }
 
       /**
-       * Add a {@link ParseNetworkInterceptor}.
-       *
-       * @param interceptor The interceptor to add.
-       * @return The same builder, for easy chaining.
-       */
-      public Builder addNetworkInterceptor(ParseNetworkInterceptor interceptor) {
-        if (interceptors == null) {
-          interceptors = new ArrayList<>();
-        }
-        interceptors.add(interceptor);
-        return this;
-      }
-
-      /**
        * Enable pinning in your application. This must be called before your application can use
        * pinning.
+       *
        * @return The same builder, for easy chaining.
        */
       public Builder enableLocalDataStore() {
         localDataStoreEnabled = true;
         return this;
       }
 
-      /* package for tests */ Builder setNetworkInterceptors(Collection<ParseNetworkInterceptor> interceptors) {
```

Comment:
Guys, be really careful when you abrubptly plan to deprecate a feature that is really useful for debugging/ troubleshooting.
Moreover, relying on an additional 3rd party library seems to be a step back from having an easy to use interceptor as this one.